base/strings/unicode.jl

```diff
@@ -318,10 +319,10 @@ Returns `true` if the given char or integer is an assigned Unicode code point.
 ```jldoctest
 julia> using Unicode
 
-julia> isassigned(101)
+julia> Unicode.isassigned(101)
```

Comment:
You don't need the module prefix after calling `using Unicode` above.

Reply:
No. It didn't work because `Unicode` doesn't either export `isassigned` or overload `Base.isassigned`. I think this is intentional.

Reply:
Ah, right, I forgot about this naming conflict. It's quite unfortunate...